src/acl/agent.test.ts

```diff
@@ -911,6 +913,101 @@ describe("setAgentResourceAccess", () => {
     );
   });
 
+  it("adds the appropriate Quads for the given Access Modes if the rule is both a resource and default rule", async () => {
+    const sourceDataset = Object.assign(dataset(), {
+      internal_accessTo: "https://arbitrary.pod/resource",
+      internal_resourceInfo: {
+        fetchedFrom: "https://arbitrary.pod/resource/?ext=acl#owner",
+        isLitDataset: true,
+      },
+    });
+    sourceDataset.add(
```

Comment:
Very clunky (and extremely verbose, and therefore hard-to-read) test data setup here. With hundreds of tests in this codebase already, we really should address this (later :) !)...

Reply:
There already are a couple of helper methods in this file for initialising mock data, perhaps we could re-use those?

Reply:
Resolved in b7e66e9be4b42d61acf1fc7decd8ef7c2461f4b8